tests/cypress/integration/filter.spec.js

```diff
@@ -0,0 +1,269 @@
+describe('app', () => {
+  // Rhino applications have an 'app' prefix
+  const nsPrefix = 'app-';
+  // Support function to help generating namespaces
+  const ns = (id) => `#${nsPrefix}${id}`;
+  // Support function to help generating teal namespaces on main UI
+  const nsTeal = (id) => ns(`teal_wrapper-teal-main_ui-${id}`);
+
+  beforeEach(() => {
+    cy.visit('/');
+
+    cy
+      .get('html')
+      .not('.shiny-busy');
+
+    // Find the tab and click to navigate
+    cy
+      .get('.nav.nav-pills a[data-bs-toggle=tab]')
+      .contains('KM plot for TTDE')
+      .as('navPills');
+
+    cy
```

Comment:
This seems useless, but it is a design pattern that avoids having detached elements in DOM